contents/docs/integrate/index.mdx

```diff
@@ -3,116 +3,54 @@ title: Integrate PostHog
--   [PHP](/docs/integrate/server/php)
--   [Ruby](/docs/integrate/server/ruby)
--   [Golang](/docs/integrate/server/go)
--   [Java](/docs/integrate/server/java)
-
-**Community libraries**
-
--   [Elixir](/docs/integrate/server/elixir)
--   [Nim](https://github.com/Yardanico/posthog-nim)
-
-Check out the pages above for information on how to set-up each library, as well as which features each library supports. _As a note, session recording is only supported in client-side libraries._
-
-## Apps
-
-Apps extend PostHog's functionality and are an incredibly powerful part of the PostHog ecosystem. If you need to bring in data from a 3rd party service or perform some operation on _all_ events coming into PostHog, apps are generally the best and most reliable option.
-
-Apps can help you:
-
--   Import data from 3rd party services
--   Enrich events in your pipeline by adding extra data
--   Filter events to keep your data clean
-
-### Installation
-
-Below is a list of all our apps for bringing data and events into PostHog. Apps can be installed from the app section in the [dashboard](https://app.posthog.com/project/apps).
-For information on how to set up each app, the list below contains links to each app's documentation page.
-
-### Ingestion apps (25)
-
-For a list of all our apps, see [here](/apps)
-
-<IngestionAppsList />
-
-> For more detail on how apps work and tutorials on how to build your own, see our [apps docs](/docs/apps).
-
-## API
-
-Events can also be ingested directly using our API and the [`/capture`](/docs/api/post-only-endpoints#capture) endpoint, which is the same endpoint that all of our libraries use behind the scenes.
-
-Generally, this isn't something you'll need to use when integrating PostHog, but if you're working with a language or framework that PostHog doesn't support yet, this will allow you to still send events.
-
-For more information, read our [API docs](/docs/api/post-only-endpoints).
+Once your PostHog instance is up and running, the next step is to start sending events.
+
+import Tab from "components/Tab"
+
+import SnippetInstructions from "./\_snippets/snippet.mdx"
+import SDKs from "./\_snippets/sdks.mdx"
+import Frameworks from "./\_snippets/frameworks.mdx"
+import API from "./\_snippets/api.mdx"
+
+<h3 class="mb-0">Installation</h3>
+
+<!-- prettier-ignore -->
+<Tab.Group>
+    <Tab.List>
+        <Tab>JS snippet</Tab>
+        <Tab count="13">Libraries</Tab>
+        <Tab count="11">Framework guides</Tab>
+        <Tab>API</Tab>
+    </Tab.List>
+    <Tab.Panels>
+        <Tab.Panel>
+            <SnippetInstructions />
+        </Tab.Panel>
+        <Tab.Panel>
+            <SDKs />
+        </Tab.Panel>
+        <Tab.Panel>
+            <Frameworks />
+        </Tab.Panel>
+        <Tab.Panel>
+            <API />
+        </Tab.Panel>
+    </Tab.Panels>
+</Tab.Group>
+
+<br />
+<hr />
+<br />
+
+### What to read next
+
+Learn about everything PostHog has to offer with these in-depth guides.
```

Comment:
This sentence makes me feel like it'll talk about features, but then it just talks about how to do more advanced integrations. Maybe...

```suggestion
Learn how to take your PostHog integrations to the next level with these in-depth guides.
```

Reply:
Good point, I like that framing better :+1:

Reply:
Background context: we try to avoid phrases like "next level" since they're vague and don't actually say anything - instead would replace with the types of things you can do.